<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderSettingsPage</class>
 <widget class="QWidget" name="OrderSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="phoneSanitizationGroupBox">
     <property name="title">
      <string>Phone number sanitization</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="phoneRemoveDashesCheckBox">
        <property name="text">
         <string>Remove dashes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="phoneRemoveSpacesCheckBox">
        <property name="text">
         <string>Remove spaces</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="phoneAddCodeCheckBox">
        <property name="text">
         <string>Add missing country code</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="phonePrefixNameLabel">
        <property name="text">
         <string>Prefix format:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="phonePrefixComboBox">
        <item>
         <property name="text">
          <string>+xx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>00xx</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>317</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
